tests/conftest.py

```diff
@@ -1596,8 +1621,28 @@ def verify_new_core_dumps(duthost):
     new_core_dumps = list(new_core_dumps)
 
     if new_core_dumps:
-        pytest.fail("Core dumps found. Expected: %s Found: %s. Test failed. New core dumps: %s" % (len(pre_existing_cores),\
-            len(cur_cores), new_core_dumps))
+        reload = True
+        logger.info("Core dumps found. Expected: %s Found: %s. New core dumps: %s" % (len(pre_existing_cores), len(cur_cores), new_core_dumps))
+
+    # Check if the running config is different from the golden config
+    if duthost.stat(path=CONFIG_DB_BAK).get('stat', {}).get('exists', False):
+        result = duthost.command("sonic-cfggen -d --print-data")
+        running_config = json.loads(result['stdout'])
+
+        result = duthost.command("cat {}".format(CONFIG_DB_BAK))
```

Comment:
This will generate another long log message in test log.

Suggest to create a new ansible module to do the config compare. Then the ansible module will be remote executed on DUT and won't generate too much test logs.

Reply:
done